### PR TITLE
Neovim installation update on dockerfile

### PR DESCRIPTION
**Pull request by neurovium**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -73,3 +73,7 @@ RUN pip install --no-cache-dir multiscale_spatial_image jupyterlab-git \
         datalad-fuse
 
 RUN pip install --no-cache-dir brainrender napari
+
+# Neovim editor
+RUN apt-get install neovim
+RUN apt-get install python3-neovim
```
Comment (satra) [inline docker/Dockerfile:79]: - neovim should be added in the apt-section near the top that uses privileged user
- python part can be installed from conda-forge

also you may want to check if neovim itself can be installed from conda forge.

Comment (satra) [reply docker/Dockerfile:79]: also equivalent setting should be applied to Dockerfile.gpu and matlab otherwise the environments will not be the same.

Comment (neurovium) [reply docker/Dockerfile]: Moved the neovim insaller to under USER $NB_USER . Removed the python (to be added later by any user using conda or pip). Did the same for Docker.gpu and Docker.matlab files.

**Comment by satra** [comment]
sorry, if i wasn't clear. apt-get cannot be under NB_USER it has to be in the root user section. see the other places where apt-get is used.

File: docker/Dockerfile [hunk 2]
```diff
@@ -33,6 +33,9 @@ RUN curl -fsSLo /usr/share/keyrings/brave-browser-archive-keyring.gpg https://br
        brave-browser \
     && rm -rf /tmp/*
 
+#Neovim
+RUN apt-get install neovim
```
Comment (satra) [inline docker/Dockerfile:37]: move to the apt-get section around line 11.

**Comment by kabilar** [comment]
Hi @neurovium, I am just catching up on this repo.  Would you be able to integrate Satra's suggestions so that we can merge?  If its no longer a priority, we can close this pull request.  Thank you.

**Comment by kabilar** [comment]
Closing as per Slack discussion with @neurovium.  Feel free to reopen if this feature is still desired.

**Pull request closed without merging**